src/components/StatusChip.tsx

```diff
@@ -0,0 +1,79 @@
+import BlockIcon from '@mui/icons-material/Block';
+import CheckIcon from '@mui/icons-material/Check';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import { Box, Typography } from '@mui/material';
+import { StandardCSSProperties } from '@mui/system';
+import { useTranslation } from 'react-i18next';
+import { NviCandidateStatus } from '../types/nvi.types';
+import { Ticket } from '../types/publication_types/ticket.types';
+
+const ticketColor = {
+  PublishingRequest: 'publishingRequest.main',
+  DoiRequest: 'doiRequest.main',
+  GeneralSupportCase: 'generalSupportCase.main',
+};
+
+interface TicketStatusChipProps {
+  ticket: Ticket;
+}
+
+export const TicketStatusChip = ({ ticket }: TicketStatusChipProps) => {
+  const { t } = useTranslation();
+
+  if (ticket.status === 'NotApplicable' || ticket.type === 'UnpublishRequest') {
+    return null;
+  }
+
+  const text = t(`my_page.messages.ticket_types.${ticket.status}`);
+
+  if (ticket.status === 'Completed') {
+    return <StatusChip text={text} icon="check" color={ticketColor[ticket.type]} />;
+  }
+
+  return <StatusChip text={text} icon={ticket.status === 'Closed' ? 'block' : 'hourglass'} />;
+};
+
+interface NviStatusChip {
+  status: NviCandidateStatus;
+}
+
+export const NviStatusChip = ({ status }: NviStatusChip) => {
+  const { t } = useTranslation();
+
+  const text = t(`tasks.nvi.status.${status}`);
+
+  if (status === 'Approved') {
+    return <StatusChip text={text} icon="check" color="nvi.main" />;
+  }
+
+  return <StatusChip text={text} icon={status === 'Rejected' ? 'block' : 'hourglass'} />;
+};
+
+interface StatusChipProps {
+  text: string;
+  icon: 'check' | 'block' | 'hourglass';
+  color?: StandardCSSProperties['backgroundColor'];
```

Comment:
```suggestion
  bgColor?: StandardCSSProperties['backgroundColor'];
```

Reply:
Nja usikker på om det trengs egentlig, for MUI sin `<Chip />` (som jeg også var innom å bruke) har også propen "color" selv om den i praksis setter bakgrunnen. I grunn samme med med Button også, men den har litt ekstra med outline, hover og sånt da

Men kan godt endre til `bgcolor` for å være eksplisitt på hvilket `sx`-felt det går til